src/components/integrated/OakLessonTopNav/OakLessonTopNav.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ReactNode } from "react";
+import styled from "styled-components";
+
+import {
+  OakBox,
+  OakFlex,
+  OakHeading,
+  OakIconProps,
+  OakSpan,
+} from "@/components/base";
+import { OakRoundIcon } from "@/components/ui";
+import { getBreakpoint } from "@/styles/utils/responsiveStyle";
+
+type LessonSectionName = "intro" | "starter-quiz" | "video" | "exit-quiz";
+
+const StyledMobileSummary = styled(OakSpan)`
+  @media (min-width: ${getBreakpoint("small")}px) {
+    display: none;
+  }
+`;
+
+export type OakLessonTopNavProps = {
+  lessonSectionName: LessonSectionName;
+  /**
+   * Slot to render `OakBackLink` or similar
+   */
+  backLinkSlot: ReactNode;
+  heading: ReactNode;
+  /**
+   * Displayed at the mobile breakpoint where the counter is not rendered.
+   * Provides alternative content for the counter/progress in the lesson.
+   */
+  mobileSummary: ReactNode;
+  /**
+   * Slot to render `OakQuizCounter` or similar
+   */
+  counterSlot?: ReactNode;
+};
+
+/**
+ * Controls for navigating back and displaying progress in a lesson
+ */
+export const OakLessonTopNav = ({
```

Comment:
I'm going to rename `OakQuizBottomNav` to `OakLessonBottomNav` in my next ticket (which is to add the lesson layout). In hindsight it didn't occur to me that the bottom nav actually applied to the whole lesson context.

I'm thinking something like:

```jsx
<OakLessonLayout topNavSlot={<OakLessonTopNav />} bottomNavSlot={<OakLessonBottomNav />}>
  current lesson content goes here
</OakLessonLayout>
```

That way in OWA `OakLessonTopNav` and `OakLessonBottomNav` can be wrapped in something that gives them access to the lesson context before being passed into their slots.